cli/cmd/install.go

```diff
@@ -244,8 +244,8 @@ func doInstallation() error {
 	}
 
 	if err := helm.NewHelper().UpgradeChart(keptnChart, keptnReleaseName, keptnNamespace, values); err != nil {
-		logging.PrintLog("Could not complete Keptn installation: "+err.Error(), logging.InfoLevel)
```

Comment:
Note: removed the `logging.Printlog` since the error message was printed twice otherwise.